osu.Game/IO/HardLinkHelper.cs

```diff
@@ -153,12 +153,12 @@ private struct ByHandleFileInformation
         public static extern int link(string oldpath, string newpath);
 
         [DllImport("libc", SetLastError = true)]
-        private static extern int stat(string pathname, out struct_stat statbuf);
+        private static extern int stat(string pathname, out Stat statbuf);
```

Comment:
For anyone else, explanation is that this triggers https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/warning-waves#cs8981---the-type-name-only-contains-lower-cased-ascii-characters.

Not sure how to feel about this change but don't think it's worth losing sleep over either.